### ISSUE TITLE
Suggestion: After sorting active mods, open a dialog listing missing dependencies

### ISSUE THREAD

**Issue by divineDerivative** [RimSort/RimSort#286]
**Is your feature request related to a problem? Please describe.**
The main feature from RimPy that is keeping me from using RimSort exclusively is the prompt for missing dependencies. In a large mod list, scrolling through the whole list to find the warning icons and then hovering over them to see the problem is tedious.

**Describe the solution you'd like**
After sorting active mods, open a dialog listing missing dependencies, and which mods need them. Allow user to add or remove any of the mods listed, and then sort the list again, or ignore the suggestions.

**Describe alternatives you've considered**
If an automatic dialog is not feasible, perhaps a menu button to open it, like 'resolve errors' or 'view missing mods'.

**Additional context**
Screenshots of the feature in RimPy
![Screenshot 2024-04-12 100150](https://github.com/RimSort/RimSort/assets/78392890/a46d5b57-cd13-4ff8-a11b-b04e61017872)
![Screenshot 2024-04-12 100224](https://github.com/RimSort/RimSort/assets/78392890/244905b3-1ffa-4a1a-beba-b33419005bb8)
![Screenshot 2024-04-12 100324](https://github.com/RimSort/RimSort/assets/78392890/769c2e85-b33d-43b1-9ef9-034ea22d57ee)
After clicking 'Apply changes and sort'
![Screenshot 2024-04-12 100423](https://github.com/RimSort/RimSort/assets/78392890/fccf0546-4d89-4861-a081-cecbd2671739)

**Comment by LionelColaso** [comment]
#89 Duplicate Issue
